packages/blockchain/bsc/src/lib/services/bsc.tx.ts

```diff
@@ -57,5 +58,8 @@ export const bscTxService = (args: { blockchain: EvmBasedBlockchain; web3: EvmBa
         smartContractApiMethod: BnbSmartChainService.bscBlockchainSmartContractInvocation,
       }),
     },
+    offchain: bscOffchainService({
```

Comment:
this should not be in txService, but as a separate service up in the main sdk - virtualACcount. Take a look to SOL or XRP / XLM